controller/tests/array_action/svc/array_mediator_svc_test.py

```diff
@@ -734,7 +734,49 @@ def test_properties(self):
         self.assertEqual(SVCArrayMediator.max_connections, 2)
         self.assertEqual(SVCArrayMediator.max_lun_retries, 10)
 
-    def _prepare_mocks_for_get_host_by_identifiers(self, svc_response, custom_host=None):
+    def _prepare_mocks_for_get_host_by_identifiers(self, nvme_host_names=None, fc_host_names=None,
+                                                   iscsi_host_name=None, connectivity_types=None):
+        host_name = 'test_host_1'
+        host_names = [host_name]
+        nvme_host_mocks = []
+        fc_host_mocks = []
+        if connectivity_types is None:
+            connectivity_types = {config.NVME_OVER_FC_CONNECTIVITY_TYPE,
+                                  config.FC_CONNECTIVITY_TYPE,
+                                  config.ISCSI_CONNECTIVITY_TYPE}
+
+        self.svc.client.svcinfo.lsnvmefabric.return_value = Mock(as_list=nvme_host_mocks)
+        if config.NVME_OVER_FC_CONNECTIVITY_TYPE in connectivity_types:
+            nvme_host_names = host_names if nvme_host_names is None else nvme_host_names
+            if nvme_host_names:
+                nvme_host_mocks = [Mock(object_name=host_name) for host_name in nvme_host_names]
+                lsnvmefabric_return_values = [Mock(as_list=[host_mock]*4) for host_mock in nvme_host_mocks]
+                self.svc.client.svcinfo.lsnvmefabric.side_effect = lsnvmefabric_return_values
```

Comment:
can you create this code in a function? `_prepare_mocks_for_get_nvme_host`
and for [fc](https://github.com/IBM/ibm-block-csi-driver/blob/ab486a6bdb3520f6fed53d00cd63895d7948aa8d/controller/tests/array_action/svc/array_mediator_svc_test.py#L756) and [iscsi](https://github.com/IBM/ibm-block-csi-driver/blob/ab486a6bdb3520f6fed53d00cd63895d7948aa8d/controller/tests/array_action/svc/array_mediator_svc_test.py#L767) accordingly

Reply:
done